bot/exts/utilities/bookmark.py

```diff
@@ -122,15 +181,46 @@ async def _bookmark_context_menu_callback(self, interaction: discord.Interaction
     @commands.guild_only()
     @whitelist_override(roles=(Roles.everyone,))
     @commands.cooldown(1, 30, commands.BucketType.channel)
-    async def bookmark(self, ctx: commands.Context) -> None:
-        """Teach the invoker how to use the new context-menu based command for a smooth migration."""
-        await ctx.send(
-            embed=self.build_error_embed(
-                "The bookmark text command has been replaced with a context menu command!\n\n"
-                "To bookmark a message simply right-click (press and hold on mobile) "
-                "on a message, open the 'Apps' menu, and click 'Bookmark'."
+    async def bookmark(
+        self,
+        ctx: commands.Context,
+        target_message: WrappedMessageConverter | None,
+        *,
+        title: str = "Bookmark",
+    ) -> None:
+        """
+        Send the author a link to the specified message via DMs.
+
+        Members can either give a message as an argument, or reply to a message.
+
+        Bookmarks can subsequently be deleted by using the `bookmark delete` command in DMs.
+        """
+        target_message: discord.Message | None = target_message or getattr(ctx.message.reference, "resolved", None)
+        if target_message is None:
+            raise commands.UserInputError(MESSAGE_NOT_FOUND_ERROR)
+
+        permissions = ctx.channel.permissions_for(ctx.author)
```

Comment:
```suggestion
        permissions = target_message.channel.permissions_for(ctx.author)
```
We care about the channel the message the user is trying to bookmark, now the channel they're running the command in.